test-connection.js

```diff
@@ -0,0 +1,9 @@
+const { sequelize } = require('./models/index');
```

Comment:
Do we need this file?

Reply:
Oh not really, i was using it to test connection when i had issue with the db connection